bayesian_cnn_prometheus/preprocessing/preprocessing_links/create_chunks.py

```diff
@@ -1,23 +1,31 @@
-from typing import Dict
+from typing import Dict, Generator
 
 import numpy as np
 
 from .chain_link import ChainLink
 
 
 class CreateChunks(ChainLink):
-    def run(self, global_config: Dict[str, str], array: np.array):
+    def run(self, global_config: Dict[str, str], array: np.array) -> Generator:
+        """
+        Creates generator to transform original data into chunks.
+        :param global_config: preprocessing config
+        :param array: image or target from the original data
+        :return: generator that produces chunks
+        """
         link_config = global_config.get('create_chunks', None)
         if self.is_activated(link_config):
             chunk_size = tuple(link_config['chunk_size'])
-            return self._create_chunks(array, chunk_size)
+            chunk_generator = self._create_chunks(array, chunk_size)
+            return chunk_generator
 
     @staticmethod
-    def _create_chunks(data_subset: np.array, chunk_size: tuple = (32, 32, 16)):
+    def _create_chunks(data_subset: np.array, chunk_size: tuple = (32, 32, 16)) -> np.array:
```

Comment:
Sam się niedawno temu dowiedziałem, ale `np.array` - to funkcja która zwraca `np.ndarray`, więc typ który powinien tutaj być - to `np.ndarray`
![image](https://user-images.githubusercontent.com/21079319/139951997-1b5ee191-bdf6-4c30-95e7-7a466b663c10.png)

Reply:
Tutaj wgl jest zwracany generator, eh